### DISCUSSION TITLE
Can't use Field with outl1ne/nova-menu-builder

### DISCUSSION THREAD

**Discussion by piotrh052b** [oneduo/nova-file-manager#117] [Q&A] [answered]
Hello, 

I am trying to use Field for selecting image with nova-menu-builder package.
Button opens select modal, but the pictures are not loading due to a 404 error.

![image](https://user-images.githubusercontent.com/64467371/210253379-a280b9be-b46c-4630-bea6-18c36c32b3da.png)

**Comment by Rezrazi** [comment]
Hi

Unfortunately we can't really help supporting all the other third party packages out there, we would really appreciate a PR for this.

**Comment by mikaelpopowicz** [comment, marked as answer]
Hi @piotrh052b 

It will be possible in a future release. You may take a look at https://github.com/oneduo/nova-file-manager/pull/127.
